.lando.yml

```diff
@@ -48,12 +46,12 @@ tooling:
     description: Run node commands
     service: node
   latest:
-    description: Download and import the latest database from Dev
+    description: Download and import the latest database from Test
```

Comment:
Is this change even necessary now that your build steps (#4) include regular DB clone from live to dev? Why not just continue to pull the DB from dev?